themes/V3/5ePHB/style.less

```diff
@@ -786,6 +786,31 @@
 // *****************************
 // *      TABLE OF CONTENTS
 // *****************************/
+
+// Default Exclusions
+h4,
+h5,
+h6,
+.page:has(.frontCover), 
+.page:has(.backCover), 
+.page:has(.insideCover), 
+.monster,
+.noToC,
+.toc { --TOC: exclude; }
+
```

Comment:
you have the label `default exclusions`, so likely best to also label the `manual inclusions` in comment.